### PR TITLE
added  my new top10flutterwidget project

### PR DESCRIPTION
**Pull request by uzairleo**
This repo has a source code of my new flutter project and also has release apk of this top10flutterwidget app. which helps those who don't know about the most useful widget of flutter. so if any wants to learn he she can easily learn from this app.

top10flutter3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ This section has projects that have just a basic UI and simple logic with locali
 
 ## Level-2: Intermediate
 - [GoogleBlessing-App](https://github.com/uzairleo/GoogleBlessings)
+- [Top10-Flutter-Widget](https://github.com/uzairleo/top10_flutter_widget)
+- [Flutter-SplashScreen-Approach](https://github.com/uzairleo/flutter_splashscreen_tutorial)
```
Comment (haroonkhan9426) [inline README.md:14]: This is a basic app, so please add it to the basic section.

Comment (uzairleo) [reply README.md:14]: sorry plz it was a mistake. i will correct it in couple of minutes

Comment (haroonkhan9426) [reply README.md:14]: Add both apps to the basic category

Comment (uzairleo) [reply README.md:14]: yes i did that with new pull request

**Pull request closed without merging**